examples/streaming_compression.c

```diff
@@ -106,17 +109,22 @@ int main(int argc, const char** argv)
 {
     const char* const exeName = argv[0];
 
-    if (argc!=2) {
+    if (argc!=3) {
```

Comment:
Nit: I'd prefer if we were able to keep the usage as `exeName FILE` with the optionality of specifying `exeName FILE LEVEL` and maybe even `exeName FILE LEVEL NUM_THREADS`.